compose.yaml

```diff
@@ -0,0 +1,25 @@
+name: unitycatalog
+
+services:
+
+  backend:
```

Comment:
s/backend/server

Reply:
Can you clarify this?

Reply:
I'd much appreciate if you could rename `backend` with `server` to give this service a bit more UC-specific flavour.